mdtraj/geometry/include/math_patch.h

```diff
@@ -1,11 +1,14 @@
 // Math functions for old versions of visual studio
-
+#ifndef __MATH_PATH_H_
+#define __MATH_PATH_H_
 #ifdef _MSC_VER
 #if _MSC_VER < 1900 // prior to vs2015
-float roundf(float x) {
+#pragma once
```

Comment:
it is probably redundant with the inline qualifier. I got trouble linking these symbols, because they were included twice. We can have another CI run without the pragma once.